### DISCUSSION TITLE
Update Vivo Type + Glypho Vivinho

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1166] [Proposals]
### Goal

Objetivo é utilizar o ícone Vivinho nos componentes de Cards 

### Context

Criando uma fonte com um ícone vivo como caracter especial nos dá a possibilidade de utilizar o ícone em qualquer lugar.

### Impact

O ícone será utilizado diretamente na fonte.

### Discussion Questions

[ ] - Validar fonte criada
[ ] - Atualizar a fonte atual Vivo Type.

Ícone utilizado = ¦ 

### Do you have a visual proposal?

https://www.figma.com/file/dWdeVR4RYaX7wOj3YAqlrx/Vivo-Type-%2B-Glypho?node-id=0-1

**Comment by yceballost** [comment]
https://github.com/Telefonica/mistica-design/issues/1171
